arch/x86/kvm/vmx/vmx_ops.h

```diff
@@ -303,6 +303,25 @@ static inline void vmcs_load(struct vmcs *vmcs)
 	vmx_asm1(vmptrld, "m"(phys_addr), vmcs, phys_addr);
 }
 
+static inline u64 vmcs_store(void)
+{
+	u64 phys_addr = INVALID_PAGE;
+
+	asm goto("1: vmptrst (%0)\n\t"
+		     ".byte 0x3e\n\t" /* branch taken hint */
```

Comment:
Does this hint make sense without a jump instruction after it?

Can we just use vmx_asm1() instead?

Reply:
Yes, it will work after we have the exception table support. The _ASM_EXTABLE uses a new exception type. This reminds me that I will add support for this new exception type in the pKVM exception handler so that it can jump to the destination set by _ASM_EXTABLE.